### PR TITLE
Add DailyOrganizer

### PR DESCRIPTION
**Pull request by mremallin**
The DailyOrganizer package provides an easy way to create daily note files and keep them organized. It also allows for additional note files to be easily created in that day's directory.

https://github.com/mremallin/DailyOrganizer

### CODE REVIEW

File: repository/d.json [hunk 1]
```diff
@@ -77,6 +77,18 @@
 				}
 			]
 		},
+		{
+			"name": "DailyOrganizer",
+			"details": "https://github.com/mremallin/DailyOrganizer",
+			"labels": ["productivity"],
+			"description": "A simple note organizer for Sublime Text.",
```
Comment (braver) [inline repository/d.json:84]: If you remove the description here it'll take that of the GitHub repo automatically, much easier to maintain.

**Comment by braver** [comment]
Please use "edit_settings" in your main menu to open the settings in split view (there are plenty of example out there).

**Comment by braver** [comment]
Please respond to the feedback to continue this PR.

File: repository/d.json (hunk 1 above)
Comment (mremallin) [reply repository/d.json]: Done

**Comment by mremallin** [comment]
> Please respond to the feedback to continue this PR.

Thanks for the comments! I've addressed in both this repo and that of the DailyOrganizer plugin. I'm sorry about the delay in getting back to you on the comments.

Cheers,
Mike.

**Comment by braver** [comment]
Nice!

**Pull request merged**